src/platform/icelake/include/platform/dai.h

```diff
@@ -55,6 +55,11 @@
 /** \brief Number of HD/A Link Inputs */
 #define DAI_NUM_HDA_IN		7
 
+/* SoundWire */
+
+/** \brief Number of SoundWire multi-gateways */
+#define DAI_NUM_SOUNDWIRE_MULTI_GTW	16
```

Comment:
I'm not sure how it is done for other interfaces, but shouldn't we define it for every cAVS platform? Right now enabling it on e.g. cnl will cause build error.

Reply:
hmm, what is a multi-gateway? This concept is not part of the hardware. The hardware is seen by the firmware as just a set of GP-DMA channels going into specific FIFOs

Reply:
@tlauda I knew I'd forget about it again...
@plbossart The plan is to aggregate multiple streams into multi-gateways like this.

Reply:
@slawblauciak the firmware is not aware of any aggregation. It does see a set of pipelines and DMA transfers going to different PDIs via the Audio Link Hub. The aggregation is controlled by the driver which enables the PDIs/Ports at the same time.